CMakeLists.txt

```diff
@@ -0,0 +1,58 @@
+cmake_minimum_required(VERSION 3.2)
+
+set(NAME_EXECUTABLE "matrix")
+set(NAME_TEST "matrix_tests")
+set(PATH_HEADERS "headers/")
+set(PATH_SRC "sources/")
+
+if(NOT DEFINED ENV{TRAVIS_BUILD_NUMBER})
+    set(ENV{TRAVIS_BUILD_NUMBER} 1)
+endif()
+
+project(${NAME_EXECUTABLE} VERSION 0.0.$ENV{TRAVIS_BUILD_NUMBER}  LANGUAGES CXX)
+
+exec_program(git ARGS submodule update --init --recursive)
```

Comment:
I think, it's better to do such things outside, somewhere in `.travis.yaml` to avoid pollution of cmake.

Reply:
Maybe you're right, I don't know which is best. It's just that this option was more convenient for me, since I worked both at home and in travis.